subgraph/src/public-lock.ts

```diff
@@ -399,3 +406,16 @@ export function handleLockMetadata(event: LockMetadataEvent): void {
     lock.save()
   }
 }
+
+export function handleReferrerFees(event: ReferrerFeeEvent): void {
+  const lock = Lock.load(event.address.toHexString())
+
+  if (lock) {
+    const hash = event.transaction.hash.toHexString()
+    const referrerFee = new ReferrerFee(hash)
```

Comment:
What happens here if the fee already exists and is just changed? We should NOT create a new object.

Reply:
Right this will make a new referrer which is not good. I fixed it here as well 👍: https://github.com/unlock-protocol/unlock/pull/12596/commits/7f641bf0176fb07ecf071031760906f8b1906121